templates/settings-admin.php

```diff
@@ -227,27 +227,46 @@ function FormatMegabytes(int $byte): string {
 
 			<div class="col col-12">
 				<div class="row">
-					<div class="col col-4 col-l-6 col-m-12">
+					<div class="col">
 						<div class="infobox">
-							<div class="interface-wrapper">
-								<?php if ($_['storage']['num_users'] > 0) : ?>
-									<?php p($l->t('Total users:')); ?>
-									<span class="info"><?php p($_['storage']['num_users']); ?></span><br>
+							<div class="interface-wrapper active-users-wrapper">
+								<?php if ($_['activeUsers']['last1hour'] > 0) : ?>
+									<div class="active-users-box">
+										<?php p($l->t('1 Hour Active Users')); ?><br>
```

Comment:
```suggestion
										<?php p($l->t('Last Hour')); ?><br>
```

Reply:
I feel like keeping the `Active users` suffix would make it more comprehensible. Maybe ask @nextcloud/designers ?